pkgs/games/blobby/default.nix

```
@@ -1,17 +1,16 @@
 {stdenv, fetchurl, SDL2, SDL2_image, libGLU, libGL, cmake, physfs, boost, zip, zlib
-, pkgconfig, unzip}:
+, pkg-config, unzip}:
 stdenv.mkDerivation rec {
-  version = "1.0";
   pname = "blobby-volley";
+  version = "1.0";
 
   src = fetchurl {
     url = "mirror://sourceforge/blobby/Blobby%20Volley%202%20%28Linux%29/1.0/blobby2-linux-1.0.tar.gz";
     sha256 = "1qpmbdlyhfbrdsq4vkb6cb3b8mh27fpizb71q4a21ala56g08yms";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [SDL2 SDL2_image libGLU libGL cmake physfs boost zip zlib
-    unzip];
+  nativeBuildInputs = [ cmake pkg-config ];
+  buildInputs = [SDL2 SDL2_image libGLU libGL physfs boost zip zlib unzip];
```

```suggestion
  buildInputs = [ SDL2 SDL2_image libGLU libGL physfs boost zip zlib unzip ];
```

I think zip and unzip could be nativeBuildInputs.

---

Just tested, looks like only `zip` is used, and I moved it into native.